src/cryptoadvance/specter/templates/wallet/send/import/wallet_importpsbt.jinja

```diff
@@ -4,13 +4,13 @@
 	{% include "includes/qr-scanner.html" %}
 	{% from 'wallet/send/components/send_nav.jinja' import send_nav %}
 	{{ send_nav('import_psbt', wallet_alias) }}
-		<h1 class="padded">{{ _("Import PSBT transaction:") }}</h1>
-		<span class="center note padded">{{ _("Paste base64-encoded PSBT transaction here to import.") }}</span>
+		<h1 class="padded">{{ _("Import transaction") }}</h1>
+		<span class="center note padded" style="margin-bottom: 10px; font-size:1em">{{ _("PSBT (base64) or old-school transaction (hex)") }}</span>
```

Comment:
"old-school" is maybe too imprecise, because for signed tx is it not "old" at all, but the default.

Reply:
"conventional" or "traditional" might be a netter term?

Reply:
You are right, @relativisticelectron. How about "serialized"?

Reply:
Yes, "serialized" is very precise.

Reply:
@k9ert :  I marked this point too early as resolved.   The https://github.com/relativisticelectron/specter-desktop/commit/3bc2f8f8dbe48f879f93969b2a99b5a12be0ba65  is after you merged the branch.